src/test/scala/edu/ie3/simona/agent/participant2/ParticipantAgentInitSpec.scala

```diff
@@ -177,11 +177,11 @@ class ParticipantAgentInitSpec
 
         participantAgent ! PrimaryRegistrationSuccessfulMessage(
           primaryService.ref,
-          15 * 3600L,
+          4 * 3600L,
```

Comment:
Why did you change this? Also at the other places in this file.

Reply:
After discussion with @danielfeismann, we decided that if data (primary and secondary) has not been received before the start of the operation interval, that's an error, and we're throwing. Beforehand, we would just postpone the start of the operation, which is something you probably wouldn't expect.